diff --git a/backend/package.json b/backend/package.json
index 6a7c4d7..336208d 100644
--- a/backend/package.json
+++ b/backend/package.json
@@ -34,7 +34,6 @@
     "@types/yamljs": "^0.2.34",
     "dotenv": "^17.3.1",
     "jest": "^30.2.0",
-    "prisma": "^7.4.0",
     "supertest": "^7.2.2",
     "ts-jest": "^29.4.6",
     "tsx": "^4.21.0",
@@ -43,7 +42,7 @@
   "dependencies": {
     "@aws-sdk/client-s3": "^3.992.0",
     "@prisma/adapter-pg": "^7.4.0",
-    "@prisma/client": "^7.4.0",
+    "@prisma/client": "^7.7.0",
     "@prisma/client-runtime-utils": "^7.4.0",
     "@types/ms": "^2.1.0",
     "bcryptjs": "^3.0.3",
@@ -57,11 +56,12 @@
     "jsonwebtoken": "^9.0.3",
     "multer": "^2.0.2",
     "multer-s3": "^3.0.1",
-    "nodemailer": "^7.0.13",
+    "nodemailer": "^8.0.5",
     "passport": "^0.7.0",
     "passport-google-oauth20": "^2.0.0",
     "passport-jwt": "^4.0.1",
     "pg": "^8.18.0",
+    "prisma": "^7.7.0",
     "swagger-ui-express": "^5.0.1",
     "yamljs": "^0.3.0",
     "zod": "^4.3.6"
diff --git a/backend/prisma.config.ts b/backend/prisma.config.ts
deleted file mode 100644
index d73df7b..0000000
--- a/backend/prisma.config.ts
+++ /dev/null
@@ -1,12 +0,0 @@
-import "dotenv/config";
-import { defineConfig, env } from "prisma/config";
-
-export default defineConfig({
-  schema: "prisma/schema.prisma",
-  migrations: {
-    path: "prisma/migrations",
-  },
-  datasource: {
-    url: env("DATABASE_URL"),
-  },
-});
diff --git a/backend/src/modules/auth/auth.service.ts b/backend/src/modules/auth/auth.service.ts
index e41f3d9..dc22687 100644
--- a/backend/src/modules/auth/auth.service.ts
+++ b/backend/src/modules/auth/auth.service.ts
@@ -31,6 +31,7 @@ export class AuthService {
 
   static async signup({ email, username, password, role }: SignupInput) {
     const existingUser = await AuthModel.findUserByEmail(email.toLowerCase());
+    console.log("=========================",existingUser);
     if (existingUser) {
       throw new CustomError(ErrorMessages.EMAIL_ALREADY_EXISTS, 409, HTTPStatusText.FAIL);
     }
